tests/testdata/codegen/convert-aliased-imports-pp/go/convert-aliased-imports.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"github.com/pulumi/pulumi-aws/sdk/v5/go/aws/ecs"
+	awsxecs "github.com/pulumi/pulumi-awsx/sdk/go/awsx/ecs"
+	"github.com/pulumi/pulumi-awsx/sdk/go/awsx/lb"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		cluster, err := ecs.NewCluster(ctx, "cluster", nil)
+		if err != nil {
+			return err
+		}
+		lb, err := lb.NewApplicationLoadBalancer(ctx, "lb", nil)
```

Comment:
Since now the `lb` variable will shadow `lb` module, will the program compilation fail if you add another resource declaration of any resource from the `lb` module after the declaration of the load balancer `lb` variable? 🤔 I am not sure we assume PCL to have already resolved variable name clashes and you don't have to fix in this PR but something to keep in mind

Reply:
that is likely true but that might cause issues in the source language to begin with? I'm not sure...for now let's keep it in mind and see if it comes up?